namespaces/aushape.yml

```diff
@@ -0,0 +1,55 @@
+
+namespace:
+  name: aushape
+  type: group
+  description: >
+    Audit events converted with aushape
+    For more information please see
+    https://github.com/Scribery/aushape
+  fields:
+  - name: serial
+    type: long
+    description: >
+      Audit event serial number
+  - name: time
+    type: date
+    description: >
+      Audit event timestamp
```

Comment:
The @timestamp field should be the date/time when the event originated at the source.  See `pipeline_metadata` to store the timestamp when the event was received at the logging collector/aggregator/etc.